.github/workflows/release.yml

```diff
@@ -1,24 +1,38 @@
----
-name: Gem Release
+name: 'Release'
```

Comment:
I thought yaml linters prefer `---` in the first line? I tried to ensure all of our workflow files contain them